src/main/webapp/WEB-INF/views/ProjectsPage.html

```diff
@@ -0,0 +1,453 @@
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <div>min</div>
+                            <div><i class="fa fa-dollar"></i>100000</div>
+                        </div>
+                        <div class="col-xs-6">
+                            <div class="right-box">Still needed</div>
+                            <div class="right-box"><i class="fa fa-dollar"></i>9999</div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+		<div class="item">
+            <div class="thumbnail">
+                <div class="caption centered">
+                    <a href="#"><h3>Project name</h3></a>
+                </div>
+                <a href="#"><img src="http://placehold.it/600x340" alt="" class="img-responsive"></a>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <div>min</div>
+                            <div><i class="fa fa-dollar"></i>100000</div>
+                        </div>
+                        <div class="col-xs-6">
+                            <div class="right-box">Still needed</div>
+                            <div class="right-box"><i class="fa fa-dollar"></i>9999</div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container">
+        <div class="row centered">
+	<ul class="pagination">
+        <li class="disabled"><a href="#">«</a></li>
+        <li class="active"><a href="#">1 <span class="sr-only">(current)</span></a></li>
+        <li><a href="#">2</a></li>
+        <li><a href="#">3</a></li>
+        <li><a href="#">4</a></li>
+        <li><a href="#">5</a></li>
+        <li><a href="#">»</a></li>
+    </ul>
+	</div>
+</div>
+<div class="footer">
+    <div class="container">
+        <div class="row centered">
+            <a href="#"><i class="fa fa-twitter"></i></a>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)